templates/project.template.yaml

```diff
@@ -86,67 +86,20 @@ Resources:
   SMSecretAlias:
     Type: AWS::KMS::Alias
     Properties:
-      AliasName: alias/secrets-manager
+      AliasName: !Sub 'alias/${AppName}-secrets-manager'
       TargetKeyId: !Ref SecretsManagerKey
 
-  Repository:
-    Type: AWS::CodeCommit::Repository
-    Properties:
-      RepositoryName: !Ref AppName
-      Triggers:
-        - CustomData: !Sub '{"SecretsManagerKey": "${SecretsManagerKey.Arn}","AppName": "${AppName}","BuildImageName": "${BuildImageName}","ArtifactBucket": "${ArtifactBucket}","ArtifactBucketKeyArn": "${ArtifactBucketKeyArn}","SamTranslationBucket": "${SamTranslationBucket}","PipelineServiceRoleArn": "${PipelineServiceRoleArn}","DevAwsAccountId": "${DevAwsAccountId}"}'
-          DestinationArn: !Ref DynamicPipelineMakerLambdaArn
-          Events:
-            - createReference
-            - deleteReference
-          Name: Create or delete dynamic pipeline for new branch
-
-  DynamicPipelineMakerLambdaPermission:
-    Type: AWS::Lambda::Permission
-    Properties: 
-      Action: lambda:InvokeFunction
-      FunctionName: !Ref DynamicPipelineMakerLambdaArn
-      Principal: codecommit.amazonaws.com
-      SourceArn: !GetAtt Repository.Arn
-
-  RepositoryUser:
-    Type: AWS::IAM::User
-    Properties:
-      UserName: !Sub AWS-CICD-Quickstart-${AppName}-${AWS::Region}-${AWS::AccountId}
-      ManagedPolicyArns:
-        - arn:aws:iam::aws:policy/AWSCodeCommitPowerUser
-
-  CodecommitCredentials:
-    Type: AWS::CloudFormation::CustomResource
+  CustomDataParameter:
+    Type: AWS::SSM::Parameter
     Properties:
-      ServiceToken: !Ref CodecommitCredentialsLambdaArn
-      IAM_USER_NAME: !Ref RepositoryUser
+      Name: !Sub '/CICD/CustomData/${FullRepositoryId}'
```

Comment:
Creating a CustomData parameter for the project, used during dynamic pipeline creation